### PR TITLE
Add automl pipeline changes to main

### PR DESCRIPTION
**Pull request by novaturient95**
# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Merge automl pipeline examples to main

fixes #

### CODE REVIEW

File: sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,922 @@
+{
```
Comment (PhaniShekhar) [inline sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb:1]: Why are we adding this notebook? We removed all standalone -mlflow notebooks. automl-image-object-detection-task-fridge-items.ipynb has all the required changes.

File: cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        max_trials: 10
+        max_concurrent_trials: 2
+      sampling_algorithm: random
+      early_termination:
+        type: bandit
+        evaluation_interval: 2
+        slack_factor: 0.2
+        delay_evaluation: 6
+    search_space:
+      - model_name: "maskrcnn_resnet50_fpn"
```
Comment (PhaniShekhar) [inline cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml:45]: This search space format is going to be changed in the September release of sdk v2. Please sync with Nilesh to get the correct format for these.

**Review by PhaniShekhar — CHANGES_REQUESTED** [review]

File: sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb (hunk 1 above)
Comment (novaturient95) [reply sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-mlflow.ipynb]: Removed. Was added erroneously, during merge.

File: cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml (hunk 2 above)
Comment (novaturient95) [reply cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml:45]: Can we can check this as part of separate PR?

**Review by harneetvirk — APPROVED** [review]

File: cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml (hunk 2 above)
Comment (sharma-riti) [reply cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml:45]: not sure if the plan is to merge this before or after the release of sdk v2.. but these examples will start failing once the new sdk release is out...

Comment (novaturient95) [reply cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml:45]: @sharma-riti Planning to have that as a separate change. Current workflow will start to fail if we add it now

**Comment by harneetvirk** [comment]
Multiple newly added files are having format issue. Please run black tool to identify those issue. Logs are here: https://github.com/Azure/azureml-examples/actions/runs/3062610889/jobs/4943779733

**Review by sharma-riti — APPROVED** [review]

**Review by PhaniShekhar — APPROVED** [review]

**Pull request merged**